DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -210,6 +210,14 @@ private function loadJackalopeSession(array $session, ContainerBuilder $containe
 
         // pipe additional parameters unchanged to jackalope
         $backendParameters += $session['backend']['parameters'];
+        if (array_key_exists('curl_options', $session['backend']) && count($session['backend']['curl_options'])) {
+            $curlOptions = array();
+            foreach ($session['backend']['curl_options'] as $option => $value) {
+                $curlOptions[constant('CURLOPT_'.$option)] = $value;
```

Comment:
is `constant()` case insensitive or do people use to write the constant names in all caps? if the later, i wonder if we should not force them to write the CURLOPT_ part themselves as well, to make things more straightforward?

should we also allow numeric options here and only resolve if the option is a string?